lib/actions/user.js

```diff
@@ -1,19 +1,17 @@
 import { createAction } from 'redux-actions'
 
 import { routeTo } from './ui'
-import {
-  addTrip,
-  addUser,
-  deleteTrip,
-  fetchUser,
-  getTrips,
-  updateTrip,
-  updateUser
-} from '../util/middleware'
+import { secureFetch } from '../util/middleware'
 import { isNewUser } from '../util/user'
 
+// Middleware API paths.
+const API_MONITORTRIP_PATH = '/api/secure/monitoredtrip'
+const API_OTPUSER_PATH = '/api/secure/user'
+const API_OTPUSER_VERIFYSMS_PATH = '/verify_sms'
+
 const setCurrentUser = createAction('SET_CURRENT_USER')
 const setCurrentUserMonitoredTrips = createAction('SET_CURRENT_USER_MONITORED_TRIPS')
+const setLastPhoneSmsRequestInfo = createAction('SET_LAST_PHONE_SMS_REQUEST_INFO')
```

Comment:
This action is never handled by a reducer. Is it needed?

Reply:
That action is used here: https://github.com/opentripplanner/otp-react-redux/blob/b1d3d7c06753a056c14bdee4c5289b222462a7e6/lib/actions/user.js#L232. Is there something else I missed?

Reply:
Oh, I totally forgot about the actual reducer file...

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/224/commits/705919a63981a453c4eceb10efd5211c547d19c1.